### ISSUE TITLE
Unexpected Data response format when calling client.getAccount

### ISSUE THREAD

**Issue by mafux777** [enigmampc/SecretJS-Templates#27]
I am using pulsar-1. 
Connecting to the node works fine.
The address I am querying has 100SCRT sitting in it (from the faucet)

Perhaps I need to update the version of secretjs?

**Comment by levackt** [comment]
Yes that's right, just update the version. Currently pulsar-1 chain is halted, but I'll retest against it and update and open [PR 26](https://github.com/scrtlabs/SecretJS-Templates/pull/26)
